Readme.md

```diff
@@ -25,30 +25,23 @@ _You need to include a link to Bootstrap 4 stylesheet in your page - these compo
 In your svelte component:
 
 ```html
+<script>
+  import { Button, Col, Row } from 'sveltestrap';
+</script>
+
 <Row>
   <Col>
     <Button color="primary" outline>Hello World!</Button>
   </Col>
 <Row>
-
-<script>
-  import { Button, Col, Row } from 'sveltestrap';
-
-  export default {
-    components: {
-      Button,
-      Col,
-      Row
-    }
-  }
-</script>
 ```
 
 ## Status
 
-* [x] Most simple, stateless components are visually done (see below).
-* [ ] Event handlers (on:click, etc), common optional attributes (id, href, data-, etc) are not applied to components! Ay!
-* [ ] Add stateful components and those that depend on tethering (e.g. Alerts, Dropdown, Modals, Popovers, Tooltips).  
+* See component list below
+* Some stateful components have issues
+  * Dropdowns all share the "open" and "closed" state. If one is open, all are open. Same with closed.
+  * Collapse uses the Svelte slide transistion. In Svelte, the component is removed from the DOM whereas in Reacstrap, it stays in the DOM, but is hidden. This interacts badly with nav bars on small screens.
```

Comment:
Does this behave correctly for you?  When I open the Collapse in the demo story the contents immediately appear, but the container's height enclosing it transitions.  I'm seeing this in latest Safari.

Reply:
If your screen is more narrow than 768px, you could be seeing the work around for nav bars.

Other than that, I didn't notice anything weird in Waterfox 56 (I should've tested more browsers, but oh well).

Reply:
This was full screen, but I only see in Safari.  Chrome is okay.  We can merge and create separate issue